Formula/swiftgen.rb

```diff
@@ -1,7 +1,7 @@
 class Swiftgen < Formula
   desc "Swift code generator for assets, storyboards, Localizable.strings, …"
   homepage "https://github.com/SwiftGen/SwiftGen"
-  url "https://github.com/SwiftGen/SwiftGen.git", :tag => "4.2.0"
+  url "https://github.com/SwiftGen/SwiftGen.git", :tag => "4.2.1"
```

Comment:
It's required for a tag, just as a checksum is required for a tarball. If the revision changes, then we contact upstream to find out if they've been hacked, and request a new version.